makeservices/easywp

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+# Creates symlink to user's webroot. If the symlink exists and files
+# are present in webroot, they are backed up in the user's home directory.
+# A user's database is created or, if it already exists, the password 
+# is reset. WordPress is installed and the wp-config file is filled
+# out with the requisite information. Further configuration can be 
+# done by the user from a browser.  
+#
+
+set -euo pipefail
+shopt -s dotglob
+
+echo -e "\033[31mWARNING: This script should generally be run on a fresh account!\033[00m"
+echo -e "A user with an existing website or database will have their files 
+backed up, but their database password will change and the WordPress 
+install may clutter or try to migrate tables.\n"
+
+read -rp "Are you sure you want to continue? [Y/n] " idunno
+idunno=${idunno,,}
+if ! [[ "$idunno" =~ ^(yes|y|^$) ]]; then
+    echo "Ok, bye!"
+    exit 0
+fi
+
+webroot=$HOME/public_html
+user=$(whoami)
+
+# Create symlink to webroot
+makehttp
+cd "$webroot"
+
+# Check if user has files and let user decide what they want to do. 
+if [[ -n "$(ls -A)" ]]; then
+    echo
+    echo -e "There are currently files in webroot!\n"
+    read -rp "Do you want to back these files up? [Y/n/x] " whoknows
```

Comment:
Why even give the choice to nuke their files? Just notify the user that they will be backed up and ask if they want to continue.